app/controllers/works_controller.rb

```diff
@@ -321,6 +326,7 @@ def handle_error_for_create(generic_error)
       end
     end
 
+    # not called
```

Comment:
In compliance with the `yard` Gem, I would please recommend that this be formatted using the following:
```
# @note not called
```

Please see https://rubydoc.info/gems/yard/file/docs/Tags.md#note for reference.